neuro_san/internals/chat/data_driven_chat_session.py

```diff
@@ -212,6 +215,9 @@ async def streaming_chat(self, user_input: str,
         # we would only be blocking our own event loop.
         await queue.put_final_item(synchronous=True)
 
+        # Close any objects on sly data that can be closed.
+        await self.close_sly_data()
```

Comment:
Now that we know we are all done, close out objects in the sly data that can be closed.